paddle/fluid/operators/detection/yolov3_loss_op.h

```diff
@@ -283,6 +297,19 @@ class Yolov3LossKernel : public framework::OpKernel<T> {
     int* gt_match_mask_data =
         gt_match_mask->mutable_data<int>({n, b}, ctx.GetPlace());
 
+    const T* gt_score_data;
+    if (!gt_score) {
+      Tensor _gt_score;
```

Comment:
_gt_score  -> gt_score , do not add _ prefix

Reply:
Done